backend/src/routes/sessions.ts

```diff
@@ -73,13 +80,28 @@ router.post(
       session.postSessionMentee = postMenteeNoteId._id;
       session.postSessionMentor = postMentorNoteId._id;
       await session.save();
+
+      await sendNotification(
+        "New session booked!",
+        "You have a new session with " + mentee.name + ". Check out your session details \u{1F60E}",
+        mentor.fcmToken
+      );
+      await sendNotification(
```

Comment:
Did the Figma mention that we should send a notification immediately after a session is booked to the Mentee? If it did then this is fine, otherwise I'm not sure it makes sense to send the notification to the mentee since they are the ones booking the event.

Reply:
Yup the figma mentioned it